docs/src/dev/principles/unsafe.md

```diff
@@ -0,0 +1,170 @@
+# Unsafe Guidance

```

Comment:
mentioned offline, but can you link this and all of the principles to the summary page so they are discoverable

Reply:
so it is, just not under the dev section as I was looking. thanks!